### PR TITLE
chore(issue-templates): improving github issue templates

### PR DESCRIPTION
**Pull request by markcaron**
Removing old templates, adding docs issue templates, and improving others

## What I did

1. Removed `Ready for Development` and `Implement an existing PatternFly` templates
2. Added `Docs problem` and `Docs edit or update request` templates
3. Improved/updated others

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 1]
```diff
@@ -10,38 +10,44 @@ body:
     attributes:
       label: Describe the bug
       description: >-
-        A clear and concise description of what the bug is
+        A clear and concise description of the bug
       placeholder: >-
         The arrow icon in the call-to-action button should animate when
         the user hovers the CTA, but it doesn't
```
Comment (hellogreg) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:16]: Add "over" after "hovers."

File: .github/ISSUE_TEMPLATE/3-feature-request.yml [hunk 2]
```diff
@@ -26,12 +27,23 @@ body:
...
+    validations:
+      required: false
+
 - type: textarea
   id: example
   attributes:
     label: Example API
     description: >-
-      Provide and example HTML snippet of how you expect the new feature might work
+     If possible, provide and example HTML snippet of how you expect the new feature might work
```
Comment (hellogreg) [inline .github/ISSUE_TEMPLATE/3-feature-request.yml:46]: Change "and" to "an."

File: .github/ISSUE_TEMPLATE/4-docs-request.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        - [Designers Get Started](https://ux.redhat.com/get-started/designers/)
+    validations:
+      required: false
+
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: >-
+        If applicable, add screenshots of examples in the wild or to help explain
```
Comment (hellogreg) [inline .github/ISSUE_TEMPLATE/4-docs-request.yml:37]: Delete "or."

**Review by zeroedin — APPROVED** [review]
Given @hellogreg well found fixes:

Legal guidelines tackle misconduct

File: .github/ISSUE_TEMPLATE/4-docs-request.yml (hunk 3 above)
Comment (markcaron) [reply .github/ISSUE_TEMPLATE/4-docs-request.yml]: Updated

File: .github/ISSUE_TEMPLATE/3-feature-request.yml (hunk 2 above)
Comment (markcaron) [reply .github/ISSUE_TEMPLATE/3-feature-request.yml]: Updated

File: .github/ISSUE_TEMPLATE/1-bug-report.yml (hunk 1 above)
Comment (markcaron) [reply .github/ISSUE_TEMPLATE/1-bug-report.yml]: Updated

**Pull request merged**